### ISSUE TITLE
Suggestion: Customize the Affiliate Registration

### ISSUE THREAD

**Issue by fitjaime** [Simplero/Roadmap#68]
In order to fully convey the benefits of becoming an affiliate, it would be awesome to have access to change the look of the Affiliate Self-Registration and Thank You for signing up pages.

Other than the logo, there isn't anything we can insert to make these more branded and cohesive with the rest of our site.

**Comment by calvincorreli** [comment]
Correct. Best bet for now would be to create a landing page that links to the registration form. Then you can explain all the benefits there, and the registration is just the registration. That's how we've done it.

**Comment by calvincorreli** [comment]
Maybe we could add an "affiliate registration" section to the new landing pages? That could be very powerful.

**Comment by Roulund** [comment]
Great idea with an "affiliate registration" section to the new landing pages.
